julia/deps/cblas.h

```diff
@@ -0,0 +1,544 @@
+#ifndef CBLAS_H
```

Comment:
Could you elaborate why this file is required? Cblas should be present inside the build environment.

Reply:
Is it necessary to have this file?

Reply:
For the Julia built form source and shipped with private OpenBLAS (e.g. Julia binary from official website), ILP64 symbol of OpenBLAS is enabled by default.
```
#ifdef INTERFACE64
# define cblas_sdsdot cblas_sdsdot64_
# define cblas_dsdot cblas_dsdot64_
# define cblas_sdot cblas_sdot64_
# define cblas_ddot cblas_ddot64_
...
```

for more info: https://github.com/dmlc/MXNet.jl/pull/267#commitcomment-24167759